docs/learn/digital-assets/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: '👋🏻 Getting Started'
+sidebar_position: 1
+description: Create a Hardhat project and start building your smart contracts for LUKSO.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started building contracts
+
+The LUKSO ecosystem offers smart contract developers a lot of [new standards](../../standards/introduction.md) and [tools](../../tools/getting-started.md) to build **powerful, modular, and standardized** blockchain applications.
+
+This page will guide you through setting up your development environment to build on LUKSO.
+
+If you need more low level information about our contracts, you can check the dedicated [contracts](../../contracts/introduction.md) section.
```

Comment:
Suggest putting that at the top as an info callout

:::info

For more low level information about the Tokens Contracts (LSP7 and LSP8), see the dedicated [**Contracts**](../../contracts/introduction.md) section.

:::